planter/README.md

```diff
@@ -28,3 +28,9 @@ Planter repects the following environment variables:
  - `PWD`: will be set to the working directory in the image
  - `USER`: used to run the image as the current user
 
+## SELinux
+
+Currently, SELinux is disabled for the container the runs the bazel
+environment, which allows for the reset of the system to leave SELinux enabled.
+Relabeling isn't done automatically since Planter mounts the entire $HOME
```

Comment:
this is true now, but we should update when https://github.com/kubernetes/test-infra/pull/6006 happens.
though even then $REPO could be `/` or something silly like that 🤷‍♂️

Reply:
I think I'll just make this generic and drop the note about "all of $HOME"

Reply:
SGTM :+1: